packages/zoltan2/src/input/Zoltan2_InputTraits.hpp

```diff
@@ -470,6 +492,7 @@ struct InputTraits<Epetra_MultiVector>
   typedef double scalar_t;
   typedef int   lno_t;
   typedef int   gno_t;
+  typedef Zoltan2::default_offset_t offset_t;
```

Comment:
If possible, let's make offset_t be size_t for all the Xpetra, Tpetra, Epetra adapters.